TECHMANIA/Assets/Scripts/Components/ResourceLoader.cs

```diff
@@ -234,7 +242,12 @@ private static bool IsEmptyFile(string fullPath)
         {
             return false;
         }
-        FileInfo fileInfo = new FileInfo(fullPath);
+        if (fullPath.StartsWith(UniversalIO.ANDROID_CONTENT_URI))
```

Comment:
IIRC this method is only called on audio files, so maybe we don't need to bypass this case?